index.bs

```diff
@@ -11299,25 +11299,26 @@ count that depends on the input channels count, the {{AudioNode}}'s
 changes.
 
 When there is a decrease in input channel count, the change in output channel
-count MUST happen after the <a>tail-time</a> has passed.
+count MUST happen when the input that was received with greater channel count no
+longer affects the output.
 
-When there is an increase in input channel count, the behavious depends on the
+When there is an increase in input channel count, the behavior depends on the
 {{AudioNode}} type:
 
 - For a {{DelayNode}} or a {{DynamicsCompressorNode}},
 	<a>tail-time</a>, the number of output channels MUST increase after the
 	<a>tail-time</a> has passed.
 
-- For the other {{AudioNode}}, since the number of input channel determines the
-	audio processing algorithm to use, the increase of the number of channels to
-	match the input channel count MUST happen immediately.
-	
+- For other {{AudioNode}}s that have a <a>tail-time</a>, the number of output
+	channels MUST increase when the input that was received with greater channel
+	count begins to affect the output.
```

Comment:
I suspect something has got confused in this merge, but I can't see how to choose the other parent for the diff in the github UI.

The "input that was received with greater channel count begins to affect the output" was intended to replace "after the tail-time has passed" above, which is what was applied at
https://github.com/WebAudio/web-audio-api/pull/1787/commits/f45873a0a8d7140508573caa9812c8c1f81aef8f
but now it is replacing the immediate change.